minishell.h

```diff
@@ -39,7 +39,7 @@
 #  define LOG_LEVEL DEBUG
 # endif
 
-extern int g_sig; // Global var for signals
+extern int g_status; // Global var exit status
```

Comment:
Not sure if this is allowed. From my understanding of the subject, the global variable is only for signals